app-devel/adaptivecpp/autobuild/defines

```diff
@@ -0,0 +1,17 @@
+PKGNAME=adaptivecpp
+PKGSEC=devel
+PKGDEP="boost gcc-runtime glibc llvm python-3"
+PKGDES="Independent, community-driven compiler for C++-based heterogeneous programming models"
```

Comment:
Compiler for C++-based heterogeneous programming models

Reply:
Fixed.